pyvrp/cpp/Individual.cpp

```diff
@@ -74,11 +82,15 @@ void Individual::evaluate(ProblemData const &data)
 
         if (static_cast<size_t>(routeLoad) > data.vehicleCapacity())
             excessLoad_ += routeLoad - data.vehicleCapacity();
+
+        uncollected_ -= routePrize;
```

Comment:
Question: should we also check that all required customers are visited? (Related: we also don't check currently that we visit each client once).

Reply:
I wasn't sure about this. On the one hand, yes, because else the solution is nonsensical. On the other hand, no, because having to check that every time could be time consuming. I'll add it in and run another benchmark later.

Reply:
This does not appear to be too expensive, so I think we can leave it in. It's a good check to have!